parts/common/components.json

```diff
@@ -343,35 +343,13 @@
         }
       ]
     },
-    {
-      "downloadURL": "mcr.microsoft.com/oss/azure/secrets-store/provider-azure:*",
-      "amd64OnlyVersions": [],
-      "multiArchVersionsV2": [
-        {
-          "renovateTag": "registry=https://mcr.microsoft.com, name=oss/azure/secrets-store/provider-azure",
-          "latestVersion": "v1.6.2"
-        }
-      ],
-      "windowsVersions": [
-        {
-          "renovateTag": "registry=https://mcr.microsoft.com, name=oss/azure/secrets-store/provider-azure",
-          "latestVersion": "v1.6.2"
-        }
-      ]
-    },
     {
       "downloadURL": "mcr.microsoft.com/oss/v2/azure/secrets-store/provider-azure:*",
       "amd64OnlyVersions": [],
       "multiArchVersionsV2": [
         {
           "renovateTag": "registry=https://mcr.microsoft.com, name=oss/v2/azure/secrets-store/provider-azure",
-          "latestVersion": "v1.7.0"
-        }
-      ],
-      "windowsVersions": [
```

Comment:
@timmy-wright as you suggested last time, I removed the `windowsVersions` field.

Reply:
Thanks - if it's the same list of versions as for Linux, then there's no point having a seperate section.